docs/graphql/core/deployment/deployment-guides/heroku.rst

```diff
@@ -42,7 +42,7 @@ Click the below button to deploy Hasura on Heroku in a few seconds:
    If you don't have an account on Heroku, you need to sign up on Heroku. You won't need a credit card, and once you
    sign up you'll be redirected to your Heroku app creation page automatically.
 
-.. thumbnail:: /img/graphql/core/guides/heroku-app.png
```

Comment:
@codeliezel we should delete this image, and not only the reference. Can you please check all deleted references and then delete those images that are nowhere used?

Reply:
@codeliezel I still can't see this image deleted